packages/ui/src/services/breakpoints/index.ts

```diff
@@ -0,0 +1,19 @@
+import { BreakpointsConfig, ThresholdsConfig } from './types'
+
+export const vaBreakpointsSymbol = Symbol('vaBreakpoints')
```

Comment:
I would suggest renaming `breakpoints` to `breakpoint` almost everywhere, similarly to how other services are implemented.

Logically, `breakpoints` would mean breakpoint list, which is not the case. Instead it's breakpoint service, which also contains other stuff.

![image](https://user-images.githubusercontent.com/5394573/190246789-58c86da3-23bb-4850-8131-c70161860237.png)